main/seccomp.c

```diff
@@ -44,6 +44,8 @@ int installSyscallFilter (void)
 	// main/parse.c:2764 : tagFilePosition (&tagfpos);
 	seccomp_rule_add (ctx, SCMP_ACT_ALLOW, SCMP_SYS (lseek), 0);
 
+	seccomp_rule_add (ctx, SCMP_ACT_ALLOW, SCMP_SYS (getpid), 0);
```

Comment:
I installed strace on travis and strace reports the final system call before ctags is terminated.
I have to spot the place where getpid is called. I hope my `-k` option is available in strace on travis.